### ISSUE TITLE
lognetwork not working anymore

### ISSUE THREAD

**Issue by Loveee** [w4kfu/TESO_fun#1]
Hi! Thanks for your fantastic work!
I wanted to try the packet decryption, but it isn't working anymore :-/

Could you be so kind and update the files for me?
Btw. is there an easier way to get plain packets? e.g. Look at a memory position and dump the plain packet? Would this possible? Then you don't have to deal with decryption.

Thanks a lot!

**Comment by w4kfu** [comment]
Hi, thanks.

Public key are not retrieved? So decryption doesn't work anymore? I don't have the hand on the executable, it's gonna be difficult to update this :/
Yes you can have decrypted packet at some point for example near the switch for each opcode.

**Comment by Loveee** [comment]
The five public keys were retrieved. I think there is maybe a change in the order which keys are choosen, like a read in a forum before? The decrypted packets don't look decrypted. I saw them before and there is nothing usable inside. At least the character stats (name,...) should be in one packet, but nothing :-/ I managed to dump the encrypted exe with olly. Now i can execute the unpacked exe on my machine, play the game without crashing. I'm only searching for a solution which realible delivers "clean" packets, that don't depend on internal code changes ;-)

But thanks a lot for your answer!

-T-
